packages/components/input/src/input.vue

```diff
@@ -327,7 +327,7 @@ useResizeObserver(textarea, (entries) => {
 const resizeTextarea = () => {
   const { type, autosize } = props
 
-  if (!isClient || type !== 'textarea') return
+  if (!isClient || type !== 'textarea' || !textarea.value) return
```

Comment:
The judgment of `textarea.value` is added here, and the non-null assertion in lines 336 and 340 can be removed.